app/assets/stylesheets/pages/course_edit_page.scss

```diff
@@ -0,0 +1,48 @@
+$white: #FFFFFF;
+
+.course {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  word-wrap: inherit;
+}
+
+.edit_text {
+  margin-top: 80px;
+  margin-bottom: 40px;
+}
+
+.section {
```

Comment:
we should be more specific about these styles - how about like edit-section? don't forget to use - instead of _